lib/segment/src/segment/entry.rs

```diff
@@ -375,6 +375,10 @@ impl SegmentEntry for Segment {
         self.id_tracker.borrow().internal_id(point_id).is_some()
     }
 
+    fn is_empty(&self) -> bool {
+        self.id_tracker.borrow().total_point_count() > 0
```

Comment:
Just want to check but this is intended, right? Naming feel as a bit off given the code.

Reply:
Yes.

Could you elaborate on why you think it's a bit off?

Reply:
:man_facepalming: Thanks!

Reply:
+1

Reply:
Patched.